src/components/app.js

```diff
@@ -185,7 +187,11 @@ const App = () => {
 
         ws.onmessage = function (event) {
             fetchConfigParameters().then((params) => {
-                dispatch(selectTheme(params.theme));
+                params.forEach((param) => {
```

Comment:
It is necessary to get all parameters ?
do not need a fetch for a single parameter ?